src/style.css

```
@@ -1 +1,68 @@
-/* Styles go here */
+html {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  padding-left: 50px;
+  padding-right: 50px;
```

Let's use shorthand for setting paddings here
```suggestion
  padding-left: 50px;
  padding-right: 50px;
```